$OpenBSD: patch-stats_c,v 1.3 2010/05/12 17:20:22 ajacoutot Exp $

Make correct use of open(2): the mode arg is only needed with the
O_CREAT flag.

--- stats.c.orig	Wed May 12 17:38:15 2010
+++ stats.c	Wed May 12 19:14:19 2010
@@ -83,7 +83,7 @@ volume_init()
    volume.is_setup = false;
 
    /* open mixer */
-   if ((volume.dev_fd = open("/dev/mixer", O_RDWR, "rw")) < 0) {
+   if ((volume.dev_fd = open("/dev/mixer", O_RDWR)) < 0) {
       warn("volume: failed to open /dev/mixer");
       return;
    }
